### ISSUE TITLE
nest desktop not running in backend 

### ISSUE THREAD

**Issue by atiye-nejad** [nest/nest-simulator#2851]
I am going to use nest desktop, I start nest server. but I can't stimulate my model and back-end nest is not running. can you help me what I have to do to activate stimulate in nest desktop.

**Comment by gtrensch** [comment]
Which installation instruction did you follow?
Please also see: https://nest-desktop.readthedocs.io/en/3.0/user/setup.html

You may also post your questions on our NEST user mailing list: https://www.nest-simulator.org/mailinglist

**Comment by atiye-nejad** [comment]
Hi, 
I had to install nest through miniconda and for nest desktop I followed instructions for the python package but I can't install it and from second step defined in site, I faced with this error:  note: This error originates from a subprocess, and is likely not a problem with pip.  ERROR: Failed building wheel for uwsgi
  Running setup.py clean for uwsgi
Failed to build uwsgi
ERROR: Could not build wheels for uwsgi, which is required to install pyproject.toml-based projects

Can I install nestdesktop through conda or  Do I have to install Docker to use Nest desktop?thanks for your support .Best, 
Atiyeh


    On Friday, July 14, 2023 at 10:09:07 AM GMT+2, Guido Trensch ***@***.***> wrote:  
 
 


Which installation instruction did you follow?
Please also see: https://nest-desktop.readthedocs.io/en/3.0/user/setup.html

You may also post your questions on our NEST user mailing list: https://www.nest-simulator.org/mailinglist

**Comment by babsey** [comment]
Hello Atiyeh,

yes, you can install `nest-desktop` via conda. 
See the detailed guide for conda environment: https://nest-desktop.readthedocs.io/en/latest/user/setup/index.html#conda-linux-windows-apple

Regarding uwsgi issue:
According to https://stackoverflow.com/questions/47868161/failed-building-wheel-for-uwsgi, you might need to install `gcc`.

Which Python version do you use?


Best regards
Sebastian

**Comment by atiye-nejad** [comment]
Hello Sebastian, 
I faced these errors using the commands written in the instructions you sent:
 Collecting package metadata (current_repodata.json): done
Solving environment: unsuccessful attempt using repodata from current_repodata.json, retrying with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed

PackagesNotFoundError: The following packages are not available from current channels:

  - nest-simulator

Current channels:

  - https://repo.anaconda.com/pkgs/main/linux-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/linux-64
  - https://repo.anaconda.com/pkgs/r/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.Best, 
Atiyeh 
   On Thursday, July 27, 2023 at 10:33:19 AM GMT+2, Sebastian Spreizer ***@***.***> wrote:  
 
 


Hello Atiyeh,

yes, you can install nest-desktop via conda.
See the detailed guide for conda environment: https://nest-desktop.readthedocs.io/en/latest/user/setup/index.html#conda-linux-windows-apple

Regarding uwsgi issue:
According to https://stackoverflow.com/questions/47868161/failed-building-wheel-for-uwsgi, you might need to install gcc.

Which Python version do you use?

Best regards
Sebastian

**Comment by babsey** [comment]
Hello Atiyeh,

did you try this (inside of conda environment)?
```
conda install -c conda-forge nest-simulator
```

**Comment by atiye-nejad** [comment]
yes, then I write these:conda install nest-desktopnest-server start
nest-desktop start
But web browser can't connect and I can't access to nest desktop. 


    On Wednesday, August 2, 2023 at 04:07:07 PM GMT+2, Sebastian Spreizer ***@***.***> wrote:  
 
 


Hello Atiyeh,

did you try this (inside of conda environment)?
conda install -c conda-forge nest-simulator

**Comment by gtrensch** [comment]
Closed due to inactivity.
